src/capi.rs

```diff
@@ -226,6 +226,34 @@ pub struct Packet {
   pub frame_type: FrameType,
 }
 
+/// Version information as presented in `[package]` `version`.
+///
+/// e.g. `0.1.0``
+///
+/// Can be parsed by [semver](https://crates.io/crates/semver).
+/// This returns the version of the loaded library, regardless
+/// of which version the library user was built against.
+pub unsafe extern fn rav1e_version_short() -> *const c_char {
```

Comment:
Both these functions appear to be missing a #[no_mangle] line. They don't show up in the generated rav1e.h without it here.